modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -24,57 +24,59 @@ ifdef::virtualmachine[]
 |Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
 endif::[]
 
-.4+|Source
-|PXE
-|Provision virtual machine from PXE menu. Requires a PXE-capable NIC in the cluster.
 
-|URL
-|Provision virtual machine from an image available from an *HTTP* or *S3* endpoint.
+|Name
+|
+|The name can contain lowercase letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
 
-|Container
-|Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
+|Template provider
+|
+|The name of the user who is creating the template for the cluster or any meaningful name that identifies this template.
 
-|Disk
-|Provision virtual machine from a disk.
+|Description
+|
+|Optional description field.
 
 |Operating System
 |
 |The primary operating system that is selected for the virtual machine.
 
-|Flavor
-|small, medium, large, tiny, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for *Flavor* are determined by the operating system.
+.4+|Boot Source
+|Import via URL (creates PVC)
+|Import content from an image available from an *HTTP* or *S3* endpoint. Example: Obtaining a URL link from the web page with the operating system image.
+
+|Clone existing PVC (creates PVC)
+|Select an existent persistent volume claim available on the cluster and clone it.
+
+|Import via Registry (creates PVC)
+|Provision virtual machine from a bootable operating system container located in a registry accessible from the cluster. Example: `_kubevirt/cirros-registry-disk-demo_`.
 
-|Memory
+|PXE (network boot - adds network interface)
+|Boot an operating system from a server on the network. Requires a PXE bootable network attachment definition.
+
+|Persistent Volume Claim project
 |
-|Size in GiB of the memory allocated to the virtual machine.
+|Select the PVC name that should apply to this virtual machine template if you cloning an existing PVC.
```

Comment:
This is for `Persistent Volume Claim name` and the below sentence `Select the project name that you want to use for cloning the PVC.` is for this(Persistent Volume Claim project)

Reply:
Corrected the text.